### DISCUSSION TITLE
Create table or column dynamically

### DISCUSSION THREAD

**Discussion by balbhushan-tech** [supabase/supabase#9556] [Questions]
I want to create table and column from my javascript code, how can i do that please guide me.

**Comment by GaryAustin1** [comment]
In general you should not dynamically create tables.  https://github.com/supabase/supabase/discussions/2981

That said, you would have to do it with SQL and so use an rpc call and function to do it.  There are hits using …Postgres use function to create tables… or similar search on how to do this. Not Supabase specific.

**Reply by balbhushan-tech** [reply]
Thank you so much for your reply
